### ISSUE TITLE
Remove summary and description from F-Droid's repo and move them here

### ISSUE THREAD

**Issue by linsui** [SecUSo/privacy-friendly-weather#158]
I just noticed that you have provided fastlane in this repo. But those metadata are also provided in F-Droid's repo so that they are overrided. I'm going to remove those metadata in F-Droid's repo in favor of the fastlane here. There are some [translations of the summary](https://gitlab.com/fdroid/fdroiddata/-/tree/master/metadata/org.secuso.privacyfriendlyweather) contributed by F-Droid contributors. Would you like to move them here? Thanks!

**Comment by sleep-yearning** [comment]
Yes that would be great to have all the info merged in this repository.
Is that something you can do, is there help required?

**Comment by linsui** [comment]
When you copy them here I'll remove them from F-Droid's repo. Other privacy friendly apps can also do the same.

**Comment by sleep-yearning** [comment]
I copied them into the repo. However, in F-Droid there is a longer english description which is not taken from our files nor was in the folder you linked. Where is that description stored and will it still overwrite our "long_description" file?

**Comment by linsui** [comment]
It's [here](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/org.secuso.privacyfriendlyweather.yml#L16). I'll remove it. Thanks!

Will you also deal with other privacy friendly apps?

**Comment by sleep-yearning** [comment]
Yes, I am happy to include the files for other PFAs as well.
I added the translations from 
- pin
- 2048
- dame
- ludo
- dicer
- notes
- ruler
- memory
- sudoku
- weather
- todolist
- paindiary
- sketching
- solitaire
- netmonitor
- cardgameone
- foodtracker
- minesweeper
- tapemeasure
- wifimanager

Is there something else to do? 
Also regarding the yml files?

**Comment by linsui** [comment]
Thanks! I'll remove those files from F-Droid's repo.

**Comment by linsui** [comment]
I removed some metadata from F-Droid's repo. https://gitlab.com/fdroid/fdroiddata/-/merge_requests/9123 Other apps' fastlanes are not covered by the tags so I kept them. When there are new tags available I'll also remove them. Thanks!

**Comment by linsui** [comment]
The filename of long description should be `full_description.txt` instead of `long_description.txt`. Could you please fix them? Thanks!

**Comment by sleep-yearning** [comment]
Oh really? That would have to be done for all the PFAs then.

**Comment by linsui** [comment]
Yep. I just notice that some PFAs in F-Droid has no description shown. Though I can't find the document except the source code https://github.com/fastlane/fastlane/search?q=full_description&type=.
